doc/source/whatsnew/v2.0.0.rst

```diff
@@ -14,6 +14,26 @@ including other versions of pandas.
 Enhancements
 ~~~~~~~~~~~~
 
+.. _whatsnew_200.enhancements.case_when:
+
+Assignment based on multiple conditions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``pd.case_when`` API has now been added to support assignment based on multiple conditions.
```

Comment:
Add a proper link to the API documentation: `` :func:`case_when` ``

Also, can you refer to this as a function rather than an API.